### PR TITLE
Complete class Robot.

### PR DESCRIPTION
**Pull request by EliosPride**

### CODE REVIEW

File: src/main/java/core/basesyntax/Robot.java [hunk 1]
```diff
@@ -1,4 +1,58 @@
...
+
+    public Robot(Direction direction, int coordinateX, int coordinateY) {
+        this.direction = direction;
+        this.coordinateX = coordinateX;
+        this.coordinateY = coordinateY;
+    }
+
+    public Direction getDirection() {
+        return direction;
+        // текущее направление взгляда
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/Robot.java]: @EliosPride remove commented lines

File: src/main/java/core/basesyntax/Robot.java [hunk 2]
```diff
@@ -1,4 +1,58 @@
...
+        // текущая координата X
+    }
+
+    public int getCoordinateY() {
+        return coordinateY;
+    }
+
+    public void turnLeft() {
+        switch (direction) {
+            case UP: direction = Direction.LEFT;
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/Robot.java]: @EliosPride missed 'break' for each case

File: src/main/java/core/basesyntax/Robot.java [hunk 3]
```diff
@@ -1,4 +1,58 @@
...
+            case DOWN: direction = Direction.LEFT;
+            case LEFT: direction = Direction.UP;
+            case RIGHT: direction = Direction.DOWN;
+        }
+        // повернуться на 90 градусов по часовой стрелке
+    }
+
+    public void stepForward() {
+        switch (direction) {
+            case UP: coordinateX += 1;
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/Robot.java]: @EliosPride use increment (++) and decrement (--)

File: src/main/java/core/basesyntax/Robot.java [hunk 4]
```diff
@@ -1,4 +1,58 @@
...
+            case DOWN: coordinateX -= 1;
+            case RIGHT: coordinateY += 1;
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/Robot.java]: @EliosPride wrong logic for stepForfard() method. Robot moves are incorrect.

File: src/main/java/core/basesyntax/Robot.java (hunk 1 above)
Comment (EliosPride) [reply src/main/java/core/basesyntax/Robot.java]: @al-pavlenko removed, done

File: src/main/java/core/basesyntax/Robot.java (hunk 2 above)
Comment (EliosPride) [reply src/main/java/core/basesyntax/Robot.java]: @al-pavlenko added 'break', done

File: src/main/java/core/basesyntax/Robot.java (hunk 3 above)
Comment (EliosPride) [reply src/main/java/core/basesyntax/Robot.java]: @al-pavlenko replace '+=', '-=', with '++', '--', done

File: src/main/java/core/basesyntax/Robot.java (hunk 4 above)
Comment (EliosPride) [reply src/main/java/core/basesyntax/Robot.java]: didn't understand why, done

File: src/main/java/core/basesyntax/Robot.java [hunk 5]
```diff
@@ -1,4 +1,76 @@
...
+            case RIGHT:
+                direction = Direction.DOWN;
+                break;
+        }
+    }
+
+    public void stepForward() {
+        switch (direction) {
+            case UP:
+                coordinateX++;
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/Robot.java]: @EliosPride It seems you have mixed up x and y coordinates :)

Comment (EliosPride) [reply src/main/java/core/basesyntax/Robot.java]: @al-pavlenko done.

**Review by al-pavlenko — APPROVED** [review]

**Pull request merged**